go/integration_test/processor/scenario.go

```diff
@@ -29,15 +29,19 @@ type Scenario struct {
 	Parameters  tosca.BlockParameters
 	Transaction tosca.Transaction
 	Receipt     tosca.Receipt
+	OperaError  error
```

Comment:
this should an implementation agnostic name, maybe `ProcessorError` or `TxError`, but not `OperaError`, wdyt?

Reply:
Our implementation will not return error for the tested cases, that's why I specifically named the field `OperaError`.